### PR TITLE
Translation ‘installing on linux’ chapter, from en version to cn

### PR DESCRIPTION
**Pull request by siu91**
Main translation below:
1. introduction
2. cluster operation
3. uninstalling

### CODE REVIEW

**Comment by Sherlock113** [comment]
Please add a space between English and Chinese characters. Thanks.

File: content/zh/docs/installing-on-linux/_index.md [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
 
-Remove KubeSphere and Kubernetes from your machines.
+从您的机器上删除KubeSphere和Kubernetes。
 
-## FAQ
+## 常见问题
 
 ### [Configure Booster for Installation](../installing-on-linux/faq/configure-booster/)
 
 Set a registry mirror to speed up downloads during installation.
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/_index.md:72]: Configure Booster for Installation：为安装配置加速器
Set a registry mirror to speed up downloads during installation. ：设置仓库镜像以加速安装时的下载速度。

**Comment by siu91** [comment]
> Please add a space between English and Chinese characters. Thanks.

I have added in another commit.

**Review by Sherlock113 — COMMENTED** [review]
About half of the pr was reviewed. Please see my comments and make the change.

For the other half, here are my suggestions:

1. Add a space between English and Chinese.
2. Do not add a space between the period of a Chinese sentence and the first Chinese character of the next sentence.
3. Please review the other half again to see if it complies with the [Localization Style Guide](https://github.com/kubesphere/website/blob/master/localization_style_guides/KubeSphere%20Localization%20Style%20Guide%20(for%20Simplified%20Chinese).md) and make necessary changes before you submit again.

Thanks.

**Comment by rayzhou2017** [comment]
already translated.

**Pull request closed without merging**